scripts/update-se-drivers.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-len */
```

Comment:
Couldn't you just split the array?

Reply:
There are 3 lines in this script which exceed max-length. With this PR I am not attempting to modify this script beyond minimum necessary to pass lint, which was added in this PR.